### PR TITLE
Support FP16/BF16 for onnxrt adaptor

### PR DESCRIPTION
**Pull request by mengniwang95**
## Type of Change

feature

## Description

Support FP16 for onnxrt adaptor

### CODE REVIEW

**Review by hshen14 — APPROVED** [review]
LGTM

**Review by hshen14 — APPROVED** [review]

File: test/mixed_precision/test_mixed_precision.py [hunk 1]
```diff
@@ -200,6 +200,31 @@ def build_pt_model():
     resnet18 = LazyImport("torchvision.models.resnet18")
     return resnet18()
 
+
+def build_yaml():
```
[the remark this answers is not in the document]
Comment (mengniwang95) [reply test/mixed_precision/test_mixed_precision.py:204]: there exists some new API examples

**Comment by chensuyue** [comment]
Pls also update this doc: https://github.com/intel/neural-compressor/blob/master/docs/source/mixed_precision.md

File: docs/source/mixed_precision.md [hunk 2]
```diff
@@ -20,18 +20,19 @@ The recently launched 3rd Gen Intel® Xeon® Scalable processor (codenamed Coope
...
+|MXNet         |&#10004;     |&#10006;     |
+
+> **During quantization, BF16 conversion can be executed if force enabled, FP16 can be executed if 'device' of config is 'gpu'. Please refer to this [document](./quantization_mixed_precision.md) for its workflow.**
 
-> **During quantization, BF16 conversion can be executed if force enabled. Please refer to this [document](./quantization_mixed_precision.md) for its workflow.**
 
 ## Get Started with Mixed Precision API
 
-To get a bf16 model, users can use the Mixed Precision API as follows.
+To get a bf16/fp16 model, users can use the Mixed Precision API as follows.
```
Comment (chensuyue) [inline docs/source/mixed_precision.md:35]: Let's add another sample for fp16 directly.

**Pull request merged**